Source/Core/VideoCommon/RenderBase.h

```diff
@@ -95,6 +99,17 @@ class Renderer
   virtual std::unique_ptr<AbstractTexture> CreateTexture(const TextureConfig& config) = 0;
   virtual std::unique_ptr<AbstractStagingTexture>
   CreateStagingTexture(StagingTextureType type, const TextureConfig& config) = 0;
+  virtual std::unique_ptr<AbstractFramebuffer>
+  CreateFramebuffer(const AbstractTexture* color_attachment,
+                    const AbstractTexture* depth_attachment) = 0;
+
+  // Framebuffer operations.
+  virtual void SetFramebuffer(const AbstractFramebuffer* framebuffer) {}
```

Comment:
I think we are also going to require an instruction to discard the old framebuffer

Reply:
Can you think of any case where we would want to discard while the framebuffer is bound, and hasn't been changed? The "set and discard" maps well to Vulkan and Metal render passes.